runtime/src/cost_model.rs

```diff
@@ -113,9 +113,9 @@ impl CostModel {
         match self.instruction_execution_cost_table.get_cost(program_key) {
             Some(cost) => *cost,
             None => {
-                let default_value = self.instruction_execution_cost_table.get_mode();
+                let default_value = self.instruction_execution_cost_table.get_default_units();
                 debug!(
-                    "Program key {:?} does not have assigned cost, using mode {}",
+                    "Instruction {:?} does not have aggregated cost, using default value {}",
```

Comment:
I think "program" was more accurate here?  The following format field is a program id and we have no way of reliably discriminating instructions in a general way

Reply:
Yea, been back-n-forth on this few times, `program` sometimes means a static code that does not associate with execution cost, until `instruction` to provide execution path, then comes `units` it'd consume. 

But pairing "Program" with program ID makes more sense.